proposals/nnnn-actors.md

```diff
@@ -663,7 +707,35 @@ When a given declaration (the "witness") satisfies a protocol requirement (the "
 * The requirement is `async`, or
 * the witness is non-isolated.
 
+An actor can satisfy an asynchronous requirement because any uses of the requirement are asynchronous, and can therefore suspend until the actor is available to execute them. Note that an actor can satisfy and asynchronous requirement with a synchronous one, in which case the normal notion of asynchronously accessing a synchronous declaration on an actor applies. For example:
+
+```swift
+protocol Server {
+  func send<Message: MessageType>(message: Message) async throws -> Message.Reply
+}
+
+actor MyServer : Server {
+  func send<Message: MessageType>(message: Message) throws -> Message.Reply { ... }  // okay, asynchronously accessed from clients of the protocol
+}
+```
+
+A non-isolated witness can always satisfy a requirement (whether synchronous or not), because the witness can run in any context---it does not need to be run on the actor because it isn't isolated to the actor.
```

Comment:
At first suprising but makes sense...

Worth an example?

```suggestion
A non-isolated witness can always satisfy a requirement (whether synchronous or not), because the witness can run in any context---it does not need to be run on the actor because it isn't isolated to the actor.

```swift
protocol Server { 
  func shutdown()
}

actor MyServer: Server { 
  nonisolated func shutdown() {
    Task.runDetached { await self.shutDownProperly() } 
  }
}
```

Or something like that?

Reply:
That's great, I'll steal it.